diff --git a/src/notification.c b/src/notification.c
index f1cc14d..6d8d83b 100644
--- a/src/notification.c
+++ b/src/notification.c
@@ -492,6 +492,10 @@ static void notification_format_message(struct notification *n)
         if (strnlen(n->msg, DUNST_NOTIF_MAX_CHARS + 1) > DUNST_NOTIF_MAX_CHARS) {
                 char * buffer = g_strndup(n->msg, DUNST_NOTIF_MAX_CHARS);
                 g_free(n->msg);
+
+                buffer[DUNST_NOTIF_MAX_CHARS-1] = '.';
+                buffer[DUNST_NOTIF_MAX_CHARS-2] = '.';
+                buffer[DUNST_NOTIF_MAX_CHARS-3] = '.';
                 n->msg = buffer;
         }
 }
@@ -559,14 +563,14 @@ void notification_update_text_to_render(struct notification *n)
                 char *new_buf;
                 if (hours > 0) {
                         new_buf =
-                            g_strdup_printf("%s (%ldh %ldm %lds old)", buf, hours,
+                            g_strdup_printf("%s\n%ldh %ldm %lds", buf, hours,
                                             minutes, seconds);
                 } else if (minutes > 0) {
                         new_buf =
-                            g_strdup_printf("%s (%ldm %lds old)", buf, minutes,
+                            g_strdup_printf("%s\n%ldm %lds", buf, minutes,
                                             seconds);
                 } else {
-                        new_buf = g_strdup_printf("%s (%lds old)", buf, seconds);
+                        new_buf = g_strdup_printf("%s\n%lds", buf, seconds);
                 }
 
                 g_free(buf);
diff --git a/src/notification.h b/src/notification.h
index 10b76ad..e3a18e5 100644
--- a/src/notification.h
+++ b/src/notification.h
@@ -8,7 +8,7 @@
 
 #include "markup.h"
 
-#define DUNST_NOTIF_MAX_CHARS 5000
+#define DUNST_NOTIF_MAX_CHARS 250
 
 enum behavior_fullscreen {
         FS_NULL,      //!< Invalid value
